### PR TITLE
CalendarService async with proper return types

### PR DESCRIPTION
**Pull request by matthewcorven**
Each public CalendarService method is async, with appropriate return type

### CODE REVIEW

**Comment by mckaragoz** [comment]
Thanks for this and sorry for late response. Did you test it manually? If you did we can merge it and publish a new release.

**Comment by matthewcorven** [comment]
> Thanks for this and sorry for late response. Did you test it manually? If you did we can merge it and publish a new release.

Yes, although I have additional contributions along the same lines. I'll update this PR.

**Comment by mckaragoz** [comment]
I also thought about changing the models property names that starts with uppercase, than we can release them both.

**Comment by matthewcorven** [comment]
All set @mckaragoz

File: GoogleApis.Blazor/Calendar/CalendarService.cs [hunk 1]
```diff
@@ -42,7 +38,7 @@ public string AccessToken
             get => _accessToken;
             set
             {
-                if (_accessToken != value)
+                if (_accessToken == value)
```
Comment (mckaragoz) [inline GoogleApis.Blazor/Calendar/CalendarService.cs:41]: I already fixed this in last release in Nuget, but it seems i didn't push the commit. Thanks.

File: GoogleApis.Blazor/Calendar/CalendarService.cs [hunk 2]
```diff
@@ -108,21 +104,21 @@ public string GetCalendars(int maxResults = 250, bool forceAccessToken = false)
...
             {
-                return contentResult;
+                return model;
             }
 
-            AccessToken = AuthService.RefreshAccessToken(_refreshToken);
-            GetCalendarById(calendarId);
-            return "";
+            AccessToken = await AuthService.RefreshAccessToken(_refreshToken);
+            return await GetCalendarById(calendarId);
```
Comment (mckaragoz) [inline GoogleApis.Blazor/Calendar/CalendarService.cs]: Its not about your PR i know, but doesn't this line may cause infinite loop? I think we should call the method again only if AccessToken is not null and different from the previous (the first access token we not refreshed with refresh token) one?

Comment (matthewcorven) [reply GoogleApis.Blazor/Calendar/CalendarService.cs]: @mckaragoz I've resolved these, thanks.

**Comment by mckaragoz** [comment]
Thanks for this great PR

**Pull request merged**